pkg/kv/kvprober/kvprober.go

```diff
@@ -306,6 +308,26 @@ func (p *Prober) Metrics() Metrics {
 	return p.metrics
 }
 
+// returnLeaseholderInfo searches through the given tracing recording for log messages
+// indicating leaseholder information, extracts leaseholder node ID,
+// and returns this information. Returns an empty string if leaseholder information
+// is not found.
+func (p *Prober) returnLeaseholderInfo(recording tracingpb.Recording) string {
```

Comment:
nit: Move this somewhere below `readProbeImpl` or similar. (Generally, public top-level functions go at the top.)

Reply:
Done!